styles/_header.scss

```diff
@@ -0,0 +1,78 @@
+
+/* HEADER */
+
+h1 {
+  a {
+    display: block;
+    letter-spacing: 1em;
+    padding-top: 70px;
+    padding-left: 1em;
+    padding-bottom: 0.5em;
+    text-indent: -9999px;
+    background-repeat: no-repeat;
+    background-position: center 70%;
+
+    .desktop & {
+      background-image: url(../img/logo_white_w320.png);
+    }
+
+    .mobile & {
+      padding-top: 30px;
+      letter-spacing: 0.6em;
+      padding-left: 0.05em;
+      background-image: url(../img/logo_w300.gif);
+    }
+  }
+
+  .mobile & {
+    margin-bottom: 0;
+  }
+
+  @media (max-width: 299px) {
+    background-size: 98%;
+  }
+}
+
+.lead {
+  font-size: 15px;
+  font-weight: 100;
+  font-family: "DOT2";
+  font-family: "Apple II", "Lucida Console", Monaco, monospace;
+  font-size: 9.5px;
+  color: #fff5c8;
+  letter-spacing: -1.5px;
+  text-transform: uppercase;
+  position: relative;
+  display: inline-block;
+  padding: 5px 8px 3px;
+  border: thin solid #fff5c8;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Using width with border can sometimes make elements larger than you expect.](https://www.codacy.com/app/j-hasseriis/cepheus/file/1863156548/issues/source?bid=3085146&fileBranchId=3086993#l48)